appium/webdriver/appium_service.py

```diff
@@ -197,7 +195,7 @@ def is_running(self) -> bool:
         """Check if the service is running.
 
         Returns:
-            bool: `True` or `False`
+            bool: `True` if the service is running
```

Comment:
Necessary here since `property`'s one doesn't have `bool` in below doc.
(I'd like to change here to `True if the service is running`, cannot do so. I know it's not good way 🤔 .)

https://appium.github.io/python-client-sphinx/webdriver.html#webdriver.appium_service.AppiumService.is_running

Reply:
ah, got it.